pkg/helpers/rates.go

```diff
@@ -1,15 +1,25 @@
 package helpers
 
 import (
+	"strconv"
+	"strings"
 	"time"
 
 	vegeta "github.com/tsenart/vegeta/v12/lib"
 )
 
-var (
-	// Default Rate
-	DefaultRate = vegeta.Rate{Freq: 5, Per: time.Second}
-
-	// Authorization Services
-	RegisterExistingClusterRate = vegeta.Rate{Freq: 25, Per: time.Second}
-)
+func ParseRate(rate string) (vegeta.Rate, error) {
```

Comment:
I believe Vegeta also parses the rate from the command-line; may be worth seeing if there's a re-usable function in the future.